sn_node/tests/common/client.rs

```diff
@@ -160,6 +162,20 @@ impl NonDroplet {
 
         Ok(local_wallet)
     }
+
+    async fn load_faucet_wallet() -> Result<LocalWallet> {
+        info!("Loading faucet...");
+        for _ in 1..6 {
+            let faucet_wallet = create_faucet_wallet();
+
+            let faucet_balance = faucet_wallet.balance();
+            if !faucet_balance.is_zero() {
+                return Ok(faucet_wallet);
+            }
+            tokio::time::sleep(Duration::from_secs(1)).await;
+        }
+        bail!("The faucet wallet is empty even after a couple retries. Did the faucet_server start correctly?");
```

Comment:
can we make the retries a const and then pass that in here, probably noting the amount of time taken too?